plutus-core/cost-model/print-cost-model/Main.hs

```diff
@@ -0,0 +1,237 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Main where
+
+{- | A program to parse a JSON representation of costing functions for Plutus Core
+   builtins and print it in readable form. -}
+
+import Data.Aeson
+import Data.Aeson.Key as Key (toString)
+import Data.Aeson.KeyMap qualified as KeyMap
+import Data.ByteString.Lazy as BSL (getContents, readFile)
+import Data.List (intercalate)
+import Data.Text (Text)
+import System.Environment (getArgs, getProgName)
+import System.Exit
+import Text.Printf (printf)
+
+data ModelComponent = Cpu | Memory
+
+defaultCostModelPath :: String
```

Comment:
do we need this? don't we already have this as a value in a module somewhere which we could use if we're using the default?

Reply:
https://github.com/input-output-hk/plutus/blob/master/plutus-core/plutus-core/src/PlutusCore/DataFilePaths.hs#L13

Reply:
> https://github.com/input-output-hk/plutus/blob/master/plutus-core/plutus-core/src/PlutusCore/DataFilePaths.hs#L13

Hmm. That doesn't actually work. It's looking for things relative to `plutus-core`, so it only works if that's the current directory.  I wrote that, but I've forgotten my reasoning now; it does at least work for the things that need it (cost model creation things, where it's always used inside TH, so I think it's only used during compilation).

Anyway, I switched to using `Paths_`.  I'm never quite sure how that works, but it does seem to work (at least when run via cabal) anywhere in the plutus repository.